pkg/template/apis/template/types.go

```diff
@@ -108,10 +108,24 @@ type TemplateInstanceSpec struct {
 // TemplateInstanceRequester holds the identity of an agent requesting a
 // template instantiation.
 type TemplateInstanceRequester struct {
-	// Username is the username of the agent requesting a template instantiation.
+	// The name that uniquely identifies this user among all active users.
```

Comment:
I'm not so sure .... look at https://github.com/kubernetes-incubator/service-catalog/pull/939/files ... the names are capitalized for example .... or can you clarify the convention you are referring to